reference/config_files.rst

```diff
@@ -28,27 +28,27 @@ This is the typical ``~/.conan/conan.conf`` file:
    compiler.version=14
    os=Windows
 
-You can configure here the path where all the packages will be stored (recomended to assign to
-some unit as X: in windows to minimize the issue of long path names)
+Here you can configure the path where all the packages will be stored (it is recomended to assign it to
+some unit, e.g. map it to X: on windows, in order to avoid having to work with long path names).
```

Comment:
Not that we dont want to work with long paths, but the issue of max 250 chars in win path names.